knative-operator/pkg/common/service_monitor.go

```diff
@@ -20,9 +20,9 @@ import (
 )
 
 const (
-	EventingBrokerServiceMonitorPath     = "deploy/resources/broker-service-monitors.yaml"
-	EventingSourceServiceMonitorPath     = "deploy/resources/source-service-monitor.yaml"
-	EventingSourcePath                   = "deploy/resources/source-service.yaml"
+	EventingBrokerServiceMonitorPath     = "deploy/resources/monitoring/broker-service-monitors.yaml"
+	EventingSourceServiceMonitorPath     = "deploy/resources/monitoring/source-service-monitor.yaml"
+	EventingSourcePath                   = "deploy/resources/monitoring/source-service.yaml"
```

Comment:
Why these changes in here? Isn't this only about serving?

Reply:
its part of the refactoring I did.

Reply:
I'll swallow this here, but generally please keep PRs with separate concerns separate. It only makes things more contentious.

Reply:
Ok I hear you.